libpdf/models/horizontal_box.py

```diff
@@ -26,13 +32,19 @@ def __init__(
         y0: float = None,
         x1: float = None,
         y1: float = None,
+        ncolor: tuple = None,
+        fontname: str = None,
     ):
         """Init the class with plain char of a character and its rectangular coordinates."""
         self.x0 = x0
         self.y0 = y0
         self.x1 = x1
         self.y1 = y1
         self.text = text
+        self.ncolor = ncolor
+        self.fontname = fontname
+        self.bold = True if "Bold" in fontname else False
```

Comment:
I think that logic can break easily or at least produce unexpected results. There is no real standard in font names, I think exposing the fontname is good enough. The bold logic can be determined by the end user (or us in case we write test cases on known PDFs). See here:
![image](https://github.com/useblocks/libpdf/assets/839934/3c9ea035-7258-46a6-b94d-f04d4ec3999f)
![image](https://github.com/useblocks/libpdf/assets/839934/5950a2cb-0b3c-4c8a-879b-bee1ebffbd40)